<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newAbout</class>
 <widget class="QDialog" name="newAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AMC4030</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>控制卡信息：</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>控制器版本： </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>生产序列号： 0x01</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Version">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Btn_Update">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>固件升级</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_AboutFuyu">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>企业官网</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ComBox_Update">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>软件信息：</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>软件版本： </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Version_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
